updating with notes for Azure AKS (#1602)
hyperledger/besu-docs: docs/public-networks/tutorials/kubernetes.md

        persistentVolumeClaim:
          claimName: teku-pvc          
```

## Azure AKS

The process for Azure is much the same as that of AWS with a couple of differences.

#### 1. Create a Network Security Group (NSG)

> Create a network security group with ports 30303 and 9000 (or equivalent) open for TCP and UDP.
> Bind this NSG to your subnet that you choose for your ethereum nodes, so that any nodes that
> are spun up have this in effect.

Reviewer: ```suggestion
Create a NSG with ports `30303` and `9000` (or equivalent) open for TCP and UDP.
Bind this NSG with the subnet you've designated for your Ethereum nodes to ensure that nodes initiated within this subnet will automatically inherit these security rules.
```

#### 2. Add a node pool to your cluster

In Azure all machines get allocated a public IP by default but you need to turn this on for your

> new nodepool

Reviewer: ```suggestion
new node pool.
```

If you are using [Terraform](https://www.terraform.io/), use something like the following for your
new node pool:

```yaml
  node_pools = {
    ...
    ethereum = {
      name                   = "ethereum"
      vm_size                = "Standard_D8as_v5"
      vnet_subnet_id         = lookup(module.vnet.vnet_subnets_name_id, "subnet-....") # The ID of the security group from the previous step.
      os_disk_size_gb        = 100
      min_count              = 1
      max_count              = 10
      enable_auto_scaling    = true
      enable_node_public_ip  = true     # This flag lets every node keep its public ip
      enable_host_encryption = true
      node_taints            = ["ethereum=true:NoSchedule", "ethereum=true:NoExecute"]
      node_labels = {
        "workloadType" = "ethereum"
      }
    }

    ...
  }
```

#### 3. Use Azure StorageClasses to suit youre needs

We recommend using either Azure Disk or Azure Files for storage for your chain data

> via the CSI storage drivers (https://learn.microsoft.com/en-us/azure/aks/csi-storage-drivers).

Reviewer: ```suggestion
using the CSI storage drivers (https://learn.microsoft.com/en-us/azure/aks/csi-storage-drivers).
```
